package.json

```diff
@@ -113,9 +114,5 @@
     "classnames": "^2.2.3",
     "raf": "^3.1.0",
     "react-pure-render": "^1.0.2"
-  },
-  "peerDependencies": {
```

Comment:
Can't do this. It can cause errors when multiple copies of React get loaded. I actually had to move `react-dom` into the `peerDependencies` specifically to fix a busted 4.6.2 build.